Webpack complains ./unstable is not exported from @fluentui/react-components package

<!--
Thanks for contacting us! We're here to help.

Before you report an issue, check if it's been reported before:

  * Search: https://github.com/microsoft/fluentui/search?type=Issues
  * Search by area or component: https://github.com/microsoft/fluentui/issues/labels

Note that if you do not provide enough information to reproduce the issue, we may not be able to take action on your report.
-->

### Environment Information

- **Package version(s)**: @fluentui/react-components@9.0.0-rc.5
- **Browser and OS versions**: N/A

### Please provide a reproduction of the bug in a codepen:

<!--
Providing an isolated reproduction of the bug in a codepen makes it much easier for us to help you. Here are some ways to get started:

  * Go to https://aka.ms/fluentpen for a starter codepen
  * You can also use the "Export to Codepen" feature for the various components in our documentation site.
  * See http://codepen.io/dzearing/pens/public/?grid_type=list for a variety of examples

Alternatively, you can also use https://aka.ms/fluentdemo to get permanent repro links if the repro occurs with an example.
(A permanent link is preferable to "use the website" as the website can change.)
-->
```ts
import { Input, Label } from '@fluentui/react-components/unstable';
```

#### Actual behavior:
The following error is reported by Webpack (v5.70.0):
```
ERROR in [js file path]
Module not found: Error: Package path ./unstable is not exported from package [root]\node_modules\@fluentui\react-components
(see exports field in [root]\node_modules\@fluentui\react-components\package.json)
 @ ./src/[js file path] 6:0-85 68:19-38
...
...
...
 @ ./src/index.ts 19:37-57
```

Now even if I replace `@fluentui/react-components/unstable` with `@fluentui/react-components/lib/unstable`, the import still won't succeed, as `lib` is not defined in `exports` field in `package.json`, either.

#### Expected behavior:

The import should succeed.

If I apply the following patch to `@fluentui/react-components/package.json`, Webpack won't have any trouble finding the `unstable` sub-package.

```diff
@@ -82,6 +82,7 @@
       "import": "./lib/index.js",
       "require": "./lib-commonjs/index.js"
     },
+    "./unstable": "./unstable",
     "./package.json": "./package.json"
   }
 }
```

### Priorities and help requested:

Are you willing to submit a PR to fix? Not yet.

Requested priority: Blocking (This blocks us from consuming v9.0.0-rc.5 and later versions of fluent UI react packages.)

Products/sites affected: (if applicable)

--- a/change/@fluentui-react-components-9e580541-4783-4fe8-9998-75fc823712e5.json
+++ b/change/@fluentui-react-components-9e580541-4783-4fe8-9998-75fc823712e5.json
@@ -1,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Adding /unstable export to exports map.",
+  "packageName": "@fluentui/react-components",
+  "email": "dzearing@microsoft.com",
+  "dependentChangeType": "patch"
+}

--- a/packages/react-components/package.json
+++ b/packages/react-components/package.json
@@ -82,6 +82,11 @@
       "import": "./lib/index.js",
       "require": "./lib-commonjs/index.js"
     },
+    "./unstable": {
+      "types": "./lib/unstable/index.d.ts",
+      "import": "./lib/unstable/index.js",
+      "require": "./lib-commonjs/unstable/index.js"
+    },
     "./package.json": "./package.json"
   }
 }

